crates/turborepo-lib/src/run/context.rs

```diff
@@ -0,0 +1,41 @@
+use std::rc::Rc;
+
+use anyhow::Result;
+use turbopath::AbsoluteSystemPathBuf;
+
+use crate::{package_json::PackageJson, run::graph::WorkspaceCatalog};
+
+pub struct Context {
```

Comment:
Can we name this `PackageGraph`? Or something like that? The `Context` name from Go is not great.